debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -568,67 +549,38 @@ static void logError(TransactionalBufferMetrics metrics, String format, Object..
      * @return size
      */
     private static int getRedoLogGroupSize(OracleConnection connection) throws SQLException {
-        return getMap(connection, SqlUtils.allOnlineLogsQuery(), getDatabaseMaxScnValue(connection)).size();
-    }
-
-    /**
-     * Get the database maximum SCN value
-     *
-     * @param connection the oracle database connection
-     * @return the maximum scn value as a string value
-     */
-    public static String getDatabaseMaxScnValue(OracleConnection connection) {
-        OracleDatabaseVersion version = connection.getOracleVersion();
-        if ((version.getMajor() == 19 && version.getMaintenance() >= 6) || (version.getMajor() > 19)) {
-            return MAX_SCN_19_6;
-        }
-        else if ((version.getMajor() == 12 && version.getMaintenance() >= 2) || (version.getMajor() > 12)) {
-            return MAX_SCN_12_2;
-        }
-        else if ((version.getMajor() == 11 && version.getMaintenance() >= 2) || (version.getMajor() == 12 && version.getMaintenance() < 2)) {
-            return MAX_SCN_11_2;
-        }
-        throw new RuntimeException("Max SCN cannot be resolved for database version " + version);
+        return connection.queryAndMap("SELECT COUNT(DISTINCT GROUP#) FROM V$LOG", (rs) -> rs.getInt(1));
```

Comment:
The method is not called anywhere?

Reply:
I'll remove it then, likely just a legacy method from the original commit.  I've tried to catch most of these as I come across them and refactor the code as it relates to the task at hand.